model/Core/Classes/DictionaryEntry.md

```diff
@@ -0,0 +1,27 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# DictionaryEntry
+
+## Summary
+
+A key with an associated value.
+
+## Description
+
+The class used for implementing a generic string mapping (also known as associative arrays, dictionaries, and hash maps) in SPDX.  Each DictionaryEntry contains a key-value pair which maps the key to its associated value.  To implement a dictionary, this class is to be used in a collection with unique keys.
```

Comment:
As an aside: I noticed you include two spaces after each period. Is there a reason for that?

Reply:
Habit.  Before [2020 when Microsoft declared 2 spaces](https://www.theverge.com/2020/4/24/21234170/microsoft-word-two-spaces-period-error-correction-great-space-debate) as incorrect, both single and double spaces were allowed.  In school, I (and millions of others) were taught that 2 spaces was correct.

Reply:
Oh wow, I didn't know about that! :O

Reply:
I'm a single-spacer myself[1], but the original reason for this  was that double-spaces emulated on the (fixed-width) typewriters the correct spacing that would be there with full movable type printing presses! Of course the only real solution in the digital era is to use TeX :grinning:

[1]: `M-: (setq sentence-end-double-space nil) RET`